### DISCUSSION TITLE
Reader mode

### DISCUSSION THREAD

**Discussion by somoso** [bromite/bromite#2314] [Ideas]
On iOS (Safari) and Firefox, there is a Reading Mode which takes an article, strips out all the ads, and only shows the text and relevant images in a single screen.

Will a feature like this make its way into Bromite?

**Comment by uazo** [comment]
![image](https://user-images.githubusercontent.com/29201891/189090479-7b9a5379-b006-4882-bfe4-9684c852bc70.png)

do you mean this?

**Comment by pabloski80** [comment]
Does it provide comparable output? I'm not quite sure. 
But as a goal, possibly they are shared, yes. Also, in firefox you can quickly toggle it by a F9 configurable shortcut or mini-icon close to address-field in address-bar. And yes it rocks in FF.
